tools/apksize/default.gradle

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * Default build settings
+ */
+android {
+    adbOptions {
+        timeOutInMs 60 * 1000
+    }
+
+    lintOptions {
+        abortOnError false
+        checkReleaseBuilds false
+    }
+    compileSdkVersion 26
+
+    defaultConfig {
+        applicationId 'com.google.apksize'
+        minSdkVersion 26
+	multiDexEnabled true
+        targetSdkVersion 26
+        versionCode 1
+        versionName '1.0'
+    }
+
+    signingConfigs {
+        debug {
+            keyAlias 'androiddebugkey'
+            keyPassword 'android'
+            storeFile file('debug.keystore')
+            storePassword 'android'
```

Comment:
Do we need to do this. If we are only using debug signing configs, we should not need to wire this in manually

Reply:
test-apps/database-test-app/build.gradle
can be a good example of a bare minimum app that we can configure

Reply:
Done.